e3sm_diags/plot/utils.py

```diff
@@ -53,6 +53,7 @@
     "meridional_mean_2d",
     "streamflow",
     "tc_analysis",
+    "aerosol_aeronet",
```

Comment:
This fixes the `aerosol_aeronet` X and Y axes labels.

Example: https://web.lcrc.anl.gov/public/e3sm/cdat-migration-fy24/25-02-13-branch-930-zppy-diffs-aerosol/viewer/aerosol_aeronet/aeronet/aodabs-global-aeronet/ann.html